editing.rst

```diff
@@ -0,0 +1,86 @@
+Editing and publishing updates to this Guide
+============================================
+
+Formatting and style
+--------------------
+While editing the Guide, please respect the existing formatting rules.
+Notably:
+
+- wrap lines at 72 characters
+- two spaces between sentences (after full stop), one space otherwise
+- one empty line between titles and paragraphs, two empty lines between
+  text and next section
+- indent using spaces, aligning to previous line
+
+When adding a new rule, use the chapter with a fitting topic.  Some
+rules match multiple chapters, choose the one that fits it best.  Make
+sure to match existing keywords in index.
+
+
+Getting sources
+---------------
+The reference repository is hosted at git.gentoo.org, and available
+via `proj/policy-guide.git gitweb`_.  To clone it::
+
+    # via https
+    git clone https://anongit.gentoo.org/git/proj/policy-guide.git
+    # via ssh
+    git clone git@git.gentoo.org:proj/policy-guide.git
+
+Non-developers wishing to send pull requests may prefer to fork
+the GitHub `gentoo/policy-guide repository`_ and clone their own fork
+instead.
+
+
+Building
+--------
+A tox-file is provided to build the Guide in a virtualenv, installing
+all necessary dependencies.  To build the HTML version, run::
+
+    tox
+
+Another format (as well as other make arguments) can be specified
+as a command-line argument, e.g.::
+
+    tox latexpdf
+
+
+Sending contributions
+---------------------
+Changes to this document can be either submitted to `Policy Manual
+Gentoo Bugzilla component`_ as git-format patches attached to the bugs,
+or to `gentoo/policy-guide repository`_ as pull requests.
+
+
+Merging pull requests
```

Comment:
This par is pointless, people can merge PRs any way they like doing it.

Reply:
It explains how to set pram for it ;-P.

Reply:
still pointless :p